closure/repositories.bzl

```diff
@@ -728,10 +728,10 @@ def com_google_template_soy():
         name = "com_google_template_soy",
```

Comment:
@sgammon is correct, `closure-templates` changed how deps are passed to the compiler, so this will break everyone using templates with dependencies. Given that it's a larger change to adapt `rules_closure`, can we revert this and only upgrade closure-library in this PR?

(On a slightly related note: Why don't we have tests for this?)

Reply:
Actually, I'm not sure why, but this doesn't seem to break `closure_{js,java}_template_library`. Apparently, we never use `--deps`.